CharlieBackend.Api/Validators/ScheduledEventDTOValidators/UpdateScheduledEventDTOValidator.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CharlieBackend.Business.Exceptions;
+using CharlieBackend.Business.Helpers;
+using CharlieBackend.Core.DTO.Schedule;
+using CharlieBackend.Data.Repositories.Impl.Interfaces;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Api.Validators.ScheduledEventDTOValidators
+{
+    public class UpdateScheduledEventDTOValidator : AbstractValidator<UpdateScheduledEventDto>
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        
+        public UpdateScheduledEventDTOValidator(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+
+            RuleFor(x => x.StudentGroupId)
```

Comment:
In my opinion, Fluent validation should call DB to validate data. It is responsibility of Service layer.
Let's discuss this on the meenting.

Reply:
Make a check that it is NotNull, MoreThan(0)

Reply:
Done